### DISCUSSION TITLE
[4.0b1] Random image folders should be restricted by filesystem plugin

### DISCUSSION THREAD

**Discussion by PhilETaylor** [joomla/joomla-cms#29490] [2. Cold Features]
### Steps to reproduce the issue

Add a new module of type random image
note that the "Image Folder" is a text field with no instructions or help

### Expected result

You should only be able to select image folders as defined in the filesystem plugin for the new media manager

### Actual result

you can type whatever you like

**Comment by adj9** [comment]
What kind of `form`do you use in the filesystem plugin?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29490">issues.joomla.org/tracker/joomla-cms/29490</a>.</sub>

**Comment by PhilETaylor** [comment]
This is not an "idea" this is a bug. Moving it to "Discussions" is PLAINLY WRONG!

**Reply by brianteeman** [reply]
Especially as you receive an email saying that the issue has been closed with zero explanation

**Reply by PhilETaylor** [reply]
Yup. But hell, don't question it, you will get attacked and told to "fix the issue yourself if its so important to you" like I did... 

So much for testing and reporting issues as being a valid contribution. I only tested Joomla 4 because I was asked to nicely by one person in leadership. This project doesn't value testing and reporting of issues. Joomla reads the Trump playbook... test less and we will have less issues to resolve... 

but I digress...

**Reply by ghost** [reply]
or you get called a troll by an "experienced team-member":

![image](https://user-images.githubusercontent.com/71960554/102683077-9abbcf00-41ce-11eb-960a-87c4f6ee6fa5.png)
